presto-docs/src/main/sphinx/connector/kinesis.rst

```diff
@@ -1,159 +1,195 @@
-    | ap-southeast-2 | Asia Pacific| (Sydney)          | kinesis.ap-southeast-2.amazonaws.com  |
-    +----------------+-------------+-------------------+---------------------------------------+
-    | ap-northeast-1 | Asia Pacific| (Tokyo)           | kinesis.ap-northeast-1.amazonaws.com  |
-    +----------------+-------------+-------------------+---------------------------------------+
-
-* ``kinesis.batch-size``: Defines maximum number of records to return in one request to Kinesis Streams. Maximum Limit is 10000 records. If a value greater than 10000 is specified, will throw InvalidArgumentException.
-  This field is optional; the default value is 10000.
-* ``kinesis.fetch-attempts``: Defines number of attempts made to read a batch from Kinesis Streams when no records are returned and the "millis behind latest" parameter shows we are not yet caught up. When records are returned no additional attempts are necessary.
-  It has been found that sometimes GetRecordResult returns empty records, when shard is not empty. That is why multiple attempts need to be made.
-  This field is optional; the default value is 2.
-* ``kinesis.max-batches``: The maximum number of batches to read in a single query. The default value is 1000.
-* ``kinesis.sleep-time``: Defines the milliseconds for which thread needs to sleep between get-record-attempts made to fetch data. The quantity should be followed by 'ms' string.
-  This field is optional; the default value is 1000ms.
-* ``iter-from-timestamp``: Use an initial shard iterator type of AT_TIMESTAMP starting iterOffsetSeconds before the current time. When this is false, an iterator type of TRIM_HORIZON will be used, meaning it will start from the oldest record in the stream.
-  The default is true.
-* ``iter-offset-seconds``: When iterFromTimestamp is true, the shard iterator will start at ``iter-offset-seconds`` before the current time.
-  The default is 86400 seconds or 24 hours.
-* ``kinesis.hide-internal-columns``: In addition to the data columns defined in a table description file, the connector maintains a number of additional columns for each table. If these columns are hidden, they can still be used in queries but do not show up in ``DESCRIBE <table-name>`` or ``SELECT *``.
-  This property is optional; the default is true.
+
+The following configuration properties are available:
+
+=================================  =======================================================================
+Property Name                      Description
+=================================  =======================================================================
+``kinesis.access-key``             Access key to aws account or blank to use default provider chain
+``kinesis.secret-key``             Secret key to aws account or blank to use default provider chain
+``kinesis.aws-region``             AWS region to be used to read kinesis stream from
+``kinesis.default-schema``	       Default schema name for tables
```

Comment:
Extra leading space

Reply:
It was actually using tabs. Not sure why IntelliJ didn't fix those.